pkg/k8s/wrapper.go

```diff
@@ -220,3 +224,59 @@ func (k *k8sWrapper) ListEvents(ops []client.ListOption) (*eventsv1.EventList, e
 	}
 	return events, nil
 }
+
+func (k *k8sWrapper) GetCNINode(cniNodeName, cniNodeNamespace string) (*rcv1alpha1.CNINode, error) {
```

Comment:
I'm not totally sold on why we need this wrapper function, as it seems more or less identical to the default behavior of the client. But if we do keep this, consider

```suggestion
func (k *k8sWrapper) GetCNINode(nn types.NamespaceName) (*rcv1alpha1.CNINode, error) {
```

Reply:
Makes sense to use namespacedname. Using this pattern to be aligned with other APIs and also easier for setup tests and call the client's API from other places.